src/devices_models/devices/AC_branches.jl

```diff
@@ -140,6 +140,37 @@ function branch_rate_constraints!(
     return
 end
 
+function branch_flow_values!(
+    optimization_container::OptimizationContainer,
+    devices::IS.FlattenIteratorWrapper{B},
+    ::DeviceModel{B, <:AbstractBranchFormulation},
+    ::Type{StandardPTDFModel},
+    ::Nothing,
+) where {B <: PSY.ACBranch}
+    ptdf = get_PTDF(optimization_container)
+    buses = ptdf.axes[2]
+    branches = ptdf.axes[1]
+    time_steps = model_time_steps(optimization_container)
+    constraint_val = JuMPConstraintArray(undef, time_steps)
+    branch_flow =
+        add_cons_container!(optimization_container, :branch_flow, branches, time_steps)
+    nodal_balance_expressions = optimization_container.expressions[:nodal_balance_active]
+    flow_variables = get_variable(optimization_container, FLOW_ACTIVE_POWER, B)
+    jump_model = get_jump_model(optimization_container)
+    for t in time_steps
+        for br in devices
+            name = PSY.get_name(br)
+            branch_flow[br_name, t] = JuMP.@constraint(
+                jump_model,
+                sum(
+                    ptdf[br_name, i] * nodal_balance_expressions[i, t] for
+                    i in ptdf.axes[2]
+                ) == flow_variables[br_name, t]
+            )
+        end
```

Comment:
Did you you extracting the loop-inner into a different function? It made a small difference for me.

Reply:
I will test that after this implementation passes the tests to see if we see the same difference.